### PR TITLE
Add fan control

### PR DESCRIPTION
**Pull request by KateLUnsworth**
Turn fan on/off if current temperature is above/below a threshold

### CODE REVIEW

File: actuators/fan_controller.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+import time
+
+import redis
+import glog
+import board
+import digitalio
+
+def parse_args():
+    """ Parse command line args to get temperature and gpio pin """
+    parser = argparse.ArgumentParser(prog="kates programe")
```
Comment (tanmaniac) [inline actuators/fan_controller.py]: prog is still kates programe pls do the needful

File: actuators/fan_controller.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    decoded_json = json.loads(message['data'])
+    temp = decoded_json["temperature_c"]
+
+    return temp
+
+def main(args):
+    """ It's a main function, what more can I say """
+    # Get args
+    gpio_pin = args.gpio_pin
+    threshold = int(args.threshold)
```
Comment (tanmaniac) [inline actuators/fan_controller.py]: Nit: in line 18, if you change `type=str` to `type=int`, you don't have to cast `threshold` here

File: actuators/fan_controller.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+    # Find pin and set up LED
+    pin = pin_mapping(args.gpio_pin)
+    led = digitalio.DigitalInOut(pin)
+    led.direction = digitalio.Direction.OUTPUT
+    
+    # Subscribe to sensor_msgs/humidity1 on redis
+    p = subscribe_pubsub("sensor_msgs/humidity1", redis_host, redis_port)
+
+    # Keep checking the temperature and turn the LED on/off based on threshold
```
Comment (tanmaniac) [inline actuators/fan_controller.py]: LED is just for testing, variable names should reference that it's a fan :wind_face:

File: actuators/fan_controller.py (hunk 1 above)
Comment (KateLUnsworth) [reply actuators/fan_controller.py]: no

**Comment by tanmaniac** [comment]
:eyes: Good code! :mount_fuji:

**Pull request merged**